src/content/docs/release-notes/logs-release-notes/logs-22-05-18.mdx

```diff
@@ -0,0 +1,34 @@
+---
+subject: Logs
+releaseDate: '2022-05-18'
+version: '220518'
+---
+
+
+import logsDashboards from 'images/logs-dashboards-052722.png'
```

Comment:
I couldn't tell if any of the guids in this image needed to be blurred out?

Reply:
No, it's Demotron. :-)